README.md

```diff
@@ -84,6 +84,18 @@ There is a rapidly expanding ecosystem surrounding Reservoir:
 - **[reservoir-docker](https://github.com/mattgrill/reservoir-docker):** Docker image for Reservoir
 - **[well](https://github.com/damontgomery/well):** Drupal installation based on Reservoir and [BLT](https://github.com/acquia/blt)
 
+
+## Drupalists
+
+If you're already deeply familiar with Drupal's concepts (such as entity types, fieldable entity types, config entity types, entity bundles, text formats, vocabularies, Views …) and you are used to finding your way around the Drupal UI, then you may find Reservoir too limiting.
+
+Reservoir limits itself to `node`s ("Content") and `node_type`s ("Content models"). These limitations make it simpler, more understandable, hence simplifying development, maintenance and the mental model.
+
+However, Reservoir still allows you to install any Drupal module to gradually add capabilities (if and when you need them): it's still Drupal under the hood. If you want the most complete Drupal experience, already preconfigured for decoupled use cases, we recommend you try [Contenta](http://www.contentacms.org/).
+
+Reservoir focuses on simplicity (but is limited). Contenta focuses on powerfulness & completeness (but has all of Drupal's copmlexity).
```

Comment:
This is based on a the discussion I had with @e0ipso yesterday, like I said in https://github.com/acquia/reservoir/pull/37#issuecomment-311984444.

Directly quoting from our IRC chat:
```
WimLeers> Reservoir is an attempt to provide a limited, focused set of API-first functionality in Drupal. Contenta wants to be more powerful and hence retains the full complexity of Drupal (but clean up e.g. normalizations).
16:54:47 <WimLeers> Reservoir only offers Nodes, Node Types, Files & Users.
16:54:47 <WimLeers> Contenta offers Nodes, Node Types, Files, Users, Comments, Comment Types, Media, Terms, Vocabularies, and so on.
```

*That* is the difference I'm getting at. It's in the paragraph above the one you quoted. I'm not referring to UI simplicity. At the same time, I'm happy to change it! What do you suggest?

Thanks for pointing out my typo :D